src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const table = document.querySelectorAll('table')[0];
+
+const tableRows = table.rows;
+
+for (let i = 0; i < tableRows.length; i++) {
+  const cellCopy = tableRows[i].cells[1];
+  const cellAfter = tableRows[i].cells[4];
```

Comment:
Ensure that each row has at least five cells to avoid potential errors when accessing `cells[1]` and `cells[4]`. Consider adding a check to verify the number of cells in each row before accessing them.